content/sensu-go/5.16/guides/using-check-aggregates.md

```diff
@@ -0,0 +1,285 @@
+---
+title: ""
+linkTitle: "Using Check Aggregates"
+description: ""
+weight: 350
+version: "5.16"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-5.16:
+    parent: guides
+---
+
+Sensu provides the capability to monitor groups of checks or entities via aggregates. In Sensu Go, this is done through the use of labels. In this tutorial, we'll walk through configuring both an entity aggregate and a check aggregate.
+
+# Entity Aggregate
+
+Let's start with configuring an entity.
+
+The first thing we'll need to do is add a label to our agent.
+
+```yaml
+---
+# Sensu agent configuration
+
+##
+# agent overview
+##
+
+labels:
+  server_type: "webservers"
+
+```
+
+In this example, we've added the label `server_type` with the value `"webservers"`--it's useful to know if a number of agents on our webservers stop reporting in rather than just a single node. Let's add a similar label to a check.
+
+# Check Aggregate
+
+The check we'll use for our aggregate is an http metric check for a ficticious app site 
+
+```yaml
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: cpu-check
+  namespace: default
+  labels:
+    app_group: "identity_servers"
+spec:
+  command: "check-http.rb -u https://identity.{{ .entity.name }}.example.com"
+  interval: 10
+  publish: true
+  handlers:
+  - slack
+  runtime_assets:
+  - sensu-plugins/sensu-plugins-http
+  - sensu/sensu-ruby-runtime
+  subscriptions:
+  - linux
+
+```
+
+While Sensu is capable of monitoring singular entities, it is also capable of monitoring groups of entities (and checks) through the use of aggregates. _______________ . Let's get started.
```

Comment:
I don't think you need this paragraph because you already introduced the idea of using aggregates to monitor groups of entities.